.github/workflows/ci-pipeline.yml

```diff
@@ -379,21 +379,21 @@ jobs:
       - if: ${{ failure() }}
         uses: ./.github/actions/run-failure-log
 
-  Run_dpdk-21_11:
-    runs-on: ubuntu-22.04
+  Run_dpdk-22_11:
```

Comment:
I guess the subject line of the commit message should start with github_ci: instead of dpdk/ci:.
The commit message could also state why the change is being made.